### ISSUE TITLE
Include prefixes

### ISSUE THREAD

**Issue by jamesmyatt** [DmytroLitvinov/awesome-flake8-extensions#129]
It would be great if this list could be augmented with a list of prefixes associated with each plugin, e.g. `flake8-bugbear -> B`.

e.g. see https://github.com/PyCQA/flake8-bugbear/issues/37#issuecomment-377075677

**Comment by orsinium** [comment]
Good idea :) I also have a little project that automatically extracts and lists all codes for many flake8 plugins:

https://flakes.orsinium.dev/

I haven't updated the list for a while, though. There are lots of new plugins that appeared since then.
